googlemock/src/gmock-spec-builders.cc

```diff
@@ -487,8 +487,31 @@ bool UntypedFunctionMockerBase::VerifyAndClearExpectationsLocked()
       // takes care of it.
       untyped_expectation->MaybeDescribeExtraMatcherTo(&ss);
       untyped_expectation->DescribeCallCountTo(&ss);
-      Expect(false, untyped_expectation->file(),
-             untyped_expectation->line(), ss.str());
+      if (! registry_destructed) {
```

Comment:
if (registry_destructed_) {
  ... do stuff...
  return;
}
... rest of function unchanged, not even indentation changes.